src/app/index.ts

```diff
@@ -18,6 +19,7 @@ const allFunctions = {
   Users: { ...UsersMod },
   Wallets: { ...WalletsMod },
   Payments: { ...PaymentsMod },
+  Swap: { ...SwapMod },
```

Comment:
wondering if this should be an app at the root function. I see that more as a service. wdyt @dolcalmi @vindard @bodymindarts ?

Reply:
ping @dolcalmi @vindard @bodymindarts

Reply:
In the context of this repository I think these are top level use cases that should be exposed via `src/app` like all other top level use cases.
Which `server` (main, daily-cron, trigger, <new-server>) ends up calling these use cases is another question.

I agree that this functionality should ideally be isolated from the 'core' backend - it belongs in the bitcoin layer - but due to coupling with the ledger (which is tightly integrated into this code base) I currently see no pragmatic alternative.

This PR is solving a short term acute pain point - so we did not set out to think about the ideal architecture for this functionality from first principles. With that background in mind I think this is correct - as it is consistent with how we are handling other similar functionalities (eg. hot -> cold wallet re-balance).

Considered in isolation with an idealised future state of our architecture in mind I would have probably advocated creating this as part of a new repository that starts bundling the bitcoin specific operational tasks - but that would be a much larger scope and create a lot more questions.